templates/questions/index.html.ep

```diff
@@ -238,21 +237,28 @@ $(document).ready(function() {
 			unanswered</div>
 		</div>
 		
-		<div class= "sortMenuTitle">
-			votes
+		<div class="sortMenuTitleSmall">
+			Show Hidden Questions
```

Comment:
Did you decide to call it hidden as opposed to downvoted or unpopular?  These would be more descriptive than "hidden".  What makes a question hidden?  Why would I want to see a hidden question?